setup.cfg

```diff
@@ -72,6 +72,7 @@ all =
     certifi
     dask[array]
     h5py
+    pyarrow>=5.0.0
```

Comment:
This adds a new optional dependency, which also should be documented in `docs/install.rst` and `docs/common_links.txt`.

Reply:
Done.